server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -71,6 +71,64 @@ public void toPlainTextShouldUpperCaseLabels() {
     }
 
     @Test
+    public void toPlainTextShouldHandleListsWell() {
+        String html = "<ul>Here is my awesome list:" +
+            "  <li>JMAP</li>" +
+            "  <li>IMAP</li>" +
+            "</ul>" +
+            "<p>Followed with some text</p>" +
+            "<p>And some other text</p>";
+        String expectedPlainText = "Here is my awesome list:  \n" +
+            " - JMAP  \n" +
+            " - IMAP\n" +
+            "\n" +
+            "Followed with some text\n" +
+            "\n" +
+            "And some other text\n" +
+            "\n";
+        assertThat(textExtractor.toPlainText(html)).isEqualTo(expectedPlainText);
+    }
+
+    @Test
+    public void tableShouldBeWellHandled() {
+        String html = " <table style=\"width:100%\">\n" +
```

Comment:
So I can't write HTML with breakline it it?

I'm forced to write HTML like this?

```
<html><head></head><body><h1>My awesome title</h1><div><p>I think nobody is compeled to write HTML like this.</p><p>This way I do not find it absurde to show we handle line breaks in HTML.</p></div></body></html>
```

And for Table it's a copy and paste from https://www.w3schools.com/html/html_tables.asp , which seems a pretty good resource <3